docs/trino-connector/trino-cascading-query.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "Apache Gravitino connector Trino cascading query"
+slug: /trino-connector/trino-cascading-query
+keyword: gravitino connector trino
+license: "This software is licensed under the Apache License version 2."
+---
+
+# Background
+With the `Apache Gravitino Trino connector` and the `Gravitino Trino cascading connector`, you can implement cascading queries in Trino.
```

Comment:
could you plz add a link for `Gravitino Trino cascading connector` here?

Reply:
fixed